plugins/kettle5-log4j-plugin/src/org/pentaho/di/core/logging/log4j/Log4jLogging.java

```diff
@@ -77,11 +73,20 @@ public void init() {
   public void dispose() {
     KettleLogStore.getAppender().removeLoggingEventListener( this );
   }
-  
-  private static void applyLog4jConfiguration() {
+
+  private void applyLog4jConfiguration() {
     LogLog.setQuietMode( true );
     LogManager.resetConfiguration();
     LogLog.setQuietMode( false );
-    DOMConfigurator.configure( PLUGIN_PROPERTIES_FILE );
+    DOMConfigurator.configure( getConfigurationFileName() );
+  }
+  
+  Logger createLogger( String loggerName ) {
```

Comment:
Package-protected methods need Javadoc to explain why they're not explicitly "protected".